docs/manual/common/concepts/ServiceDiscovery.md

```diff
@@ -1,33 +1,51 @@
-# Service Discovery
+# Registering and discovering services
 
-Microservices must support location transparency: the ability to reside on multiple hosts for load balancing and resilience, to move from one host to another when failures occur, and to scale the number of hosts up and down as load changes. 
+To be resilient and scalable, a system must support location transparency.  This allows you to run instances of the same microservice on multiple hosts, to move instances from one host to another when failures occur, and to scale the number of hosts up and down as load changes. In such a reactive system where microservice instances keep moving, spawning and dying, both clients and other microservices need a way to locate available service instances. 
 
-If a microservice's instances keep moving, spawning and dying, a mechanism for the microservice client to locate an instance is required. A Service Registry maintains an up-to-date list of available microservices that includes the host and port for each instance of each microservice. As load changes, the system can spawn or destroy instances in any location and continue to satisfy requests because the Service Registry collaborates with the microservice's instances to maintain the lookup table updated.
+Lagom provides the following functionality for your microservice systems:
 
-In order to register every microservice in the Service Registry there are two alternatives: self-registration and [3rd Party Registration](http://microservices.io/patterns/3rd-party-registration.html). When booting a Lagom microservice instance a registrar will register the name of the microservice, the URL and the names of the locatable Service Descriptors on the Service Registry so that they can be located. When powering down an instance of a service, the registrar will have to update the Service Registry too. Lagom's [[Developer Environment|DevEnvironment]] provides an implementation of a Service Registry and a registrar so you can run your microservices locally.
+* [Service registration](#Service-registration)
+* [Client-side service discovery](#Client-side-service-discovery)
+* [Server-side service discovery](#Server-side-service-discovery)
 
-There are plenty of technologies providing Service Registry capabilities. You will need to choose and/or develop a Service Locator for your services to run in your deployment environment (see for example [Lagom ZooKeeper Service Locator](https://github.com/jboner/lagom-service-locator-zookeeper)). You may need to work out a way to plug your Lagom services with a registrar. Lagom's integration with [[ConductR|ConductR]] makes this two steps seamless.
+*Note*: This page uses HTTP-REST in the examples but the concepts apply to any kind of traffic such as plain HTTP and binary over tcp.
```

Comment:
I think that's inherited from my original sentence: "-This section uses HTTP-REST for all the examples but the concepts apply to any kind of traffic be it HTTP, binary over tcp, etc..." which was slightly different and still as misleading.

In the context of presenting Service Discovery this sentence could go from:
 * *Note*: This page uses HTTP-REST in the examples but the concepts apply to any kind of traffic such as plain HTTP and binary over tcp.
 * *Note*: This page uses HTTP calls in the examples but the concepts apply to any kind of network traffic such binary over tcp.